client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -190,6 +198,24 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 
 		return (
 			<div>
+				<FormFieldset>
+					<FormLabel className="registrant-extra-info__optional" htmlFor="organization">
+						{ translate( 'Organization Name' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **15** times:
`translate( 'Add Organization Name' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/cctlds/add-organization-to-tld-forms)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👎  The "Add" makes a pretty big difference to the meaning in this case